test/run-test.ps1

```diff
@@ -98,19 +87,17 @@ Get-ChildItem -Recurse -Filter Dockerfile |
         if ($platform -eq "linux") {
             $selfContainedImage = "self-contained-build-${buildImage}"
             Write-Host "----- Creating publish-image for self-contained app built on $fullSdkTag -----"
-            exec {
-                (Get-Content ${dockerFilesPath}Dockerfile.linux.publish).Replace("{image}", $buildImage) `
-                    | docker build $optionalDockerBuildArgs -t $selfContainedImage -
+            exec {(Get-Content ${testFilesPath}Dockerfile.linux.publish).Replace("{image}", $buildImage) `
```

Comment:
Consider putting a space after the opening curly brace to be consistent with the other `exec` calls.